app/src/main/java/com/kickstarter/viewmodels/projectpage/ProjectPageViewModel.kt

```diff
@@ -655,6 +665,23 @@ interface ProjectPageViewModel {
                 }
                 .addToDisposable(disposables)
 
+            viewModelScope.launch {
```

Comment:
Now, we need to add tests on `ProjectPageViewModelTest`, as a reminder in it mandatory to test any new logic added to a VM unless a really strong reason for why not to do so appears.

also a nitpick: on line 681: no need to call `environment.apolloClientV2()`, there is a variable  on line 294.
<img width="828" alt="Screenshot 2025-06-03 at 3 46 59 PM" src="https://github.com/user-attachments/assets/5ca18934-63c5-4fa9-90f3-97c1724e0321" />